language/move-lang/tests/move_check/typing/constant_non_base_type.exp

```diff
@@ -55,9 +55,3 @@ error[E04012]: invalid type for constant
   │               Unpermitted constant type
   │               Found: '(_, _)'. But expected one of: 'u8', 'u64', 'u128', 'bool', 'address', 'vector<_>'
 
-error[E04013]: invalid statement or expression in constant
```

Comment:
Why did this stop being an error? If this is intentional, it seems like it should be in a separate test instead of being grouped with 3 of things that are expected to fail.

Reply:
Ah, I see there was a test case added to `constant_supported_exps.move`. I think it should be removed from here, though.

Reply:
well... maybe. The test still needs to see that you can't do `const C: (u64, u64) = (0, 1);` That type is still disallowed, but the expression is now allowed. There is no way of really using it though outside of letbindings, so I doubt anyone will even notice/care

Reply:
Got it. I had missed that the previous error is for this same line of code.